examples/jsm/webxr/VRButton.js

```diff
@@ -71,7 +71,7 @@ var VRButton = {
 					// ('local' is always available for immersive sessions and doesn't need to
 					// be requested separately.)
 
-					var sessionInit = { optionalFeatures: [ 'local-floor', 'bounded-floor' ] };
+					var sessionInit = { optionalFeatures: [ 'local-floor', 'bounded-floor', 'hand-tracking' ] };
```

Comment:
This is important to mention as adding `hand-tracking` by default could have performance implications as the device could be looking for hands constantly.
I didn't find a way to add options to VRButton as they were deprecated
@Mugen87 @mrdoob ideas?

Reply:
Well, we could bring `options` back. When setting the reference space was refactored, the object had no purpose anymore and thus was removed.

Reply:
Let's wait for the answer here to see if it's worth to do it optional or not https://twitter.com/fernandojsg/status/1286968102476029953

Reply:
On Firefox Reality for Hololens there's IPC involved so once you enable hand input we _have_ to calculate and send over hand poses each frame. I haven't perceived a major perf impact on simple demos but it might not be great for complex ones.

It might be worth unconditionally requesting hands in _VR_ so that you can render a hand model, but I think it might be good for an opt in. But it's worth measuring on both Oculus and FxR.

Reply:
It is the same for oculus.

Reply:
It seems like the IPC/compute overhead would be enough to make it opt in, let alone the current pose GC issue (which we both hope to fix 😉)